docs/man_pages/project/testing/livesync-ios.md

```diff
@@ -3,13 +3,14 @@ livesync ios
 
 Usage | Synopsis
 ------|-------
-General | `$ tns livesync ios [--device <Device ID>] [--watch]`
+General | `$ tns livesync ios [--device <Device ID>] [--watch] [--emulator]`
```

Comment:
Can you have --device and --emulator simultaneously? If not, split this into a few rows:

```
Synchronize changes to all connected devices | `$ tns livesync ios [--watch]`
Synchronize changes to a selected device | `$ tns livesync ios --device <Device ID> [--watch]`
Synchronize changes to the iOS Simulator | `$ tns livesync ios --emulator [--watch]`
```